src/views/pane/series-markers-pane-view.ts

```diff
@@ -151,13 +155,22 @@ export class SeriesMarkersPaneView implements IUpdatablePaneView {
 
 		return this._autoScaleMargins;
 	}
-	protected _hasAllMarkerSamePosition(): string|null {
-		const markers = this._series.indexedMarkers();
-		const markersPositions = markers.every((i: InternalSeriesMarker<TimePointIndex>) => i.position === markers[0].position);
-		if (markersPositions) {
-			return markers[0].position;
+
+	protected _getMarkerPositions(): MarkerPositions {
+		if (this._markersPositions === null) {
+			this._markersPositions = this._series.indexedMarkers().reduce((acc: MarkerPositions, marker: InternalSeriesMarker<TimePointIndex>) => {
+				if (!acc[marker.position]) {
+					acc[marker.position] = true;
+				}
+				return acc;
+				// eslint-disable-next-line @typescript-eslint/tslint/config
+			}, {
+				inBar: false,
+				aboveBar: false,
+				belowBar: false,
+			});
```

Comment:
Everything looks good except this `// eslint-disable-next-line @typescript-eslint/tslint/config`. I guess it's a formatting issue. Try to change the code to

```suggestion
			this._markersPositions = this._series.indexedMarkers().reduce(
				(acc: MarkerPositions, marker: InternalSeriesMarker<TimePointIndex>) => {
					if (!acc[marker.position]) {
						acc[marker.position] = true;
					}
					return acc;
				},
				{
					inBar: false,
					aboveBar: false,
					belowBar: false,
				}
			);
```

Reply:
Thanks